ruby-v2/Gemfile

```
@@ -1,4 +1,5 @@
-# frozen_string_literal: true
-source 'https://rubygems.org'
+source "https://rubygems.org"
+
+# See https://github.com/algolia/algoliasearch-client-ruby 
+gem 'algolia', git: 'https://github.com/algolia/algoliasearch-client-ruby.git', tag: 'v2.0.0-alpha.1'
```

```suggestion
gem 'algolia', git: 'https://github.com/algolia/algoliasearch-client-ruby.git', tag: '2.0.0-alpha.4'
```

Using the latest [alpha tag](https://github.com/algolia/algoliasearch-client-ruby/tree/2.0.0-alpha.4)